corgi/core/models.py

```diff
@@ -1012,6 +1012,7 @@ def released_components(self, include: bool = True) -> "ComponentQuerySet":
             software_build__relations__type__in=(
                 ProductComponentRelation.Type.ERRATA,
                 ProductComponentRelation.Type.COMPOSE,
+                ProductComponentRelation.Type.APP_INTERFACE,
```

Comment:
I don't think this is a good idea. We don't want these to accidentally make their way to the customer portal, as it's RH internal information. Griffon should work without them and meet the 'manifest' requirement that way.

In other words instead of having a flat 'SBOM' for each managed service, the requirement should be that the api/v1/components?ofuri=< managed service ofuri > returns the correct components for the service.